### PR TITLE
A temp PR for leaving notes about Azure:azure-iot-sdk-csharp#367

### PR DESCRIPTION
**Pull request by damonbarry**

### CODE REVIEW

File: device/Microsoft.Azure.Devices.Client.PCL/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+﻿using System.Resources;
```
Comment (damonbarry) [inline device/Microsoft.Azure.Devices.Client.PCL/Properties/AssemblyInfo.cs]: This file shouldn't be here...the PCL stuff was removed in master. I'll take care of it...

File: common/src/service/AuthenticationMethodFactory.cs [hunk 2]
```diff
@@ -13,11 +13,15 @@ public sealed class AuthenticationMethodFactory
     {
         static internal IAuthenticationMethod GetAuthenticationMethod(IotHubConnectionStringBuilder iotHubConnectionStringBuilder)
         {
+            if (!string.IsNullOrWhiteSpace(iotHubConnectionStringBuilder.DeviceId))
```
Comment (varunpuranik) [inline common/src/service/AuthenticationMethodFactory.cs:16]: Do we need an equivalent for ModuleId?

File: device/Microsoft.Azure.Devices.Client/DeviceClient.cs [hunk 3]
```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
+﻿    // Copyright (c) Microsoft. All rights reserved.
```
Comment (varunpuranik) [inline device/Microsoft.Azure.Devices.Client/DeviceClient.cs]: Nit: Unintentional tab?

File: device/tests/Microsoft.Azure.Devices.Client.Test.UWP/project.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+﻿{
```
Comment (damonbarry) [inline device/tests/Microsoft.Azure.Devices.Client.Test.UWP/project.json]: This was also deleted from master. I'll delete it here too...

File: device/devdoc/requirements/deviceclient_requirements.md [hunk 5]
```diff
@@ -66,8 +66,17 @@ public sealed class DeviceClient
...
-    public async Task SetMethodHandlerAsync(string methodName, MethodCallback methodHandler, object userContext)
+    public async Task SendEventAsync(string outputName, Message message);
+    public async Task SendEventBatchAsync(string outputName, IEnumerable<Message> messages);
+    public async Task SetEventHandlerAsync(string inputName, MessageHandler messageHandler, object userContext);
+    public async Task SetEventDefaultHandlerAsync(MessageHandler messageHandler, object userContext);
+
+    internal async Task OnReceiveEventMessageCalled(EventMessageInternal eventMessageInternal);
 
+    public async Task<Twin> GetTwinAsync();
+    public Task<MethodResult> InvokeMethodAsync(string id, Method method);
```
Comment (varunpuranik) [inline device/devdoc/requirements/deviceclient_requirements.md]: My bad, but I think we need to remove these InvokeMethod calls from here, for now at least.

File: device/Microsoft.Azure.Devices.Client/DeviceClient.cs (hunk 3 above)
Comment (damonbarry) [reply device/Microsoft.Azure.Devices.Client/DeviceClient.cs]: Yes. Will fix...

File: common/src/service/AuthenticationMethodFactory.cs (hunk 2 above)
Comment (damonbarry) [reply common/src/service/AuthenticationMethodFactory.cs:16]: This is service SDK code...I don't think we support connecting to the service endpoints as a module yet...do we? It looks we're missing a lot of code we'd need to do that. For example, there's no `ServiceAuthenticationWithModuleCredentials` class.

Comment (varunpuranik) [reply common/src/service/AuthenticationMethodFactory.cs:16]: Ah yes, makes sense. Thanks.

File: device/devdoc/requirements/deviceclient_requirements.md (hunk 5 above)
Comment (damonbarry) [reply device/devdoc/requirements/deviceclient_requirements.md]: Easy. I'll do it...

**Comment by damonbarry** [comment]
Azure/azure-iot-sdk-csharp#367 is approved and going through the checkin gate, so closing this PR.

**Pull request closed without merging**